ui/app/models/ssh/ca-config.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Model, { attr } from '@ember-data/model';
+import { expandAttributeMeta } from 'vault/utils/field-to-attrs';
+
+export default class SshCaConfig extends Model {
+  @attr('string') backend; // dynamic path of secret -- set on response from value passed to queryRecord
+  @attr('string', { sensitive: true }) privateKey; // obfuscated, never returned by API
+  @attr('string', { sensitive: true }) publicKey;
+  @attr('boolean', {
+    defaultValue: true,
+  })
```

Comment:
nit: I think it's nice when these are on the same line if there's only a single param. definitely a very pick nit 😆 
```suggestion
  @attr('boolean', { defaultValue: true })
```